### ISSUE TITLE
Error: 'JavaInfo' value has no field or method 'transitive_deps'

### ISSUE THREAD

**Issue by sgowroji** [protocolbuffers/protobuf#13829]
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/3279#018a58cb-0932-4de9-bf67-7a18fad66141

Platform : MacOS, Linux

Logs: 
```
ERROR: /Users/buildkite/builds/bk-imacpro-13/bazel-downstream-projects/protobuf/java/kotlin/BUILD.bazel:31:15: in kt_jvm_library rule //java/kotlin:only_for_use_in_proto_generated_code_its_generator_and_tests:
Traceback (most recent call last):
	File "/private/var/tmp/_bazel_buildkite/ad84fd5efc93b8e709b74b860badb45e/external/io_bazel_rules_kotlin/kotlin/internal/jvm/impl.bzl", line 216, column 36, in kt_jvm_library_impl
		_kt_jvm_produce_jar_actions(ctx, "kt_jvm_library") if ctx.attr.srcs else export_only_providers(
	File "/private/var/tmp/_bazel_buildkite/ad84fd5efc93b8e709b74b860badb45e/external/io_bazel_rules_kotlin/kotlin/internal/jvm/compile.bzl", line 491, column 50, in kt_jvm_produce_jar_actions
		outputs_struct = _run_kt_java_builder_actions(
	File "/private/var/tmp/_bazel_buildkite/ad84fd5efc93b8e709b74b860badb45e/external/io_bazel_rules_kotlin/kotlin/internal/jvm/compile.bzl", line 729, column 32, in _run_kt_java_builder_actions
		_run_merge_jdeps_action(
	File "/private/var/tmp/_bazel_buildkite/ad84fd5efc93b8e709b74b860badb45e/external/io_bazel_rules_kotlin/kotlin/internal/jvm/compile.bzl", line 264, column 70, in _run_merge_jdeps_action
		inputs = depset(jdeps, transitive = [depset([], transitive = [dep.transitive_deps for dep in deps])])
Error: 'JavaInfo' value has no field or method 'transitive_deps'
Available attributes: _compile_time_java_dependencies, _constraints, _neverlink, _transitive_full_compile_time_jars, annotation_processing, api_generating_plugins, compilation_info, compile_jars, full_compile_jars, java_outputs, module_flags_info, outputs, plugins, runtime_output_jars, source_jars, transitive_compile_time_jars, transitive_native_libraries, transitive_runtime_jars, transitive_source_jars
(04:08:33) ERROR: /Users/buildkite/builds/bk-imacpro-13/bazel-downstream-projects/protobuf/java/kotlin/BUILD.bazel:31:15: Analysis of target '//java/kotlin:only_for_use_in_proto_generated_code_its_generator_and_tests' failed
(04:08:33) ERROR: Analysis of target '//java/kotlin-lite:test_lite_extensions' failed; build aborted: Analysis failed
```
Culprit : https://github.com/bazelbuild/bazel/commit/72058be0628e84741232cb116a7199eb92638f84

CC Greenteam @fweikert

**Comment by sgowroji** [comment]
https://github.com/protocolbuffers/protobuf/pull/13830

**Comment by sgowroji** [comment]
Closed at cl/563772250
